cumulusci/utils/parallel/task_worker_queues/tests/test_parallel_worker.py

```diff
@@ -343,9 +343,9 @@ def test_worker__cannot_move_to_outdir(self):
 # Also we usually mock refresh_oauth_token which is what would
 # invoke this. In other words, using integration tests to cover this
 # function is far easier than using unit test.
-class TestSubprocessKeyChain:
+class TestSubprocessKeychain:
```

Comment:
Ideally this test would move into a test module in the same package as the code being tested, but it hardly seems worth the effort.